### PR TITLE
Path: Feature/axis map dressup (also fixes issue 2553)

### PR DESCRIPTION
**Pull request by mlampert**
PR includes 3 features/bugfixes:
* importing g-code correctly processes G20/G21 commands
* correct display of rotational parameters, https://www.freecadweb.org/tracker/view.php?id=2553
* a new dressup to map a planar axis onto a rotational axis

### CODE REVIEW

File: src/Mod/Path/PathScripts/PathDressupAxisMap.py [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+
+
+
+class ObjectDressup:
+
+    def __init__(self, obj):
+        maplist = ["X->A", "Y->A", "X->B", "Y->B", "X->C", "Y->C"]
+        obj.addProperty("App::PropertyLink", "Base","Path", "The base path to modify")
+        obj.addProperty("App::PropertyEnumeration", "axisMap", "Path", "The input mapping axis")
+        obj.addProperty("App::PropertyDistance", "radius", "Path", "The radius of the wrapped axis")
```
Comment (luzpaz) [inline src/Mod/Path/PathScripts/PathDressupAxisMap.py]: @sliptonic do these need to be wrapped in`translate()`?

**Comment by yorikvanhavre** [comment]
merged

**Comment by wwmayer** [comment]
Please have a look at the warnings on travis caused by the changes in ViewProviderPath.cpp.

**Comment by mlampert** [comment]
I'll check it out later today, thanks for the ping.
--
Sent from myMail for Android Sunday, 13 May 2018, 09:27AM -07:00 from wwmayer  notifications@github.com :

**Pull request closed without merging**